gap/DocumentationTree.gi

```diff
@@ -178,22 +186,26 @@ InstallMethod( DocumentationTree, [ ],
 end );
 
 ## create a chapter, section or subsection
-InstallMethod( StructurePartInTree, [ IsTreeForDocumentation, IsList ],
+## This first helper, given tree and context, returns a list of two entries.
```

Comment:
Insert an empty line before this? Or is this intentionally joined to the preceding comment?